File: src/Moryx.CommandCenter.Web/src/databases/models/DatabaseConfigModel.ts
/*
 * Copyright (c) 2026 Phoenix Contact GmbH & Co. KG
 * Licensed under the Apache License, Version 2.0
*/

import Entry from "../../modules/models/Entry";

export default class DatabaseConfigModel {
    public configuratorType: string;
    public connectionString: string;
    public properties: Entry;
}

File: src/Moryx.Runtime.Endpoints/Databases/Models/DatabaseConfigModel.cs
// Copyright (c) 2025, Phoenix Contact GmbH & Co. KG
// Licensed under the Apache License, Version 2.0

using Moryx.Serialization;

namespace Moryx.Runtime.Endpoints.Databases.Models
{
    /// <summary>
    /// Configuration of the database.
    /// </summary>
    public record DatabaseConfigModel
    {
        /// <summary>
        /// Name of the database configurator type
        /// </summary>
        public string ConfiguratorType { get; set; }

        /// <summary>
        /// Configuration of the database model.
        /// </summary>
        public Entry Properties { get; set; }

        public string ConnectionString { get; set; }
    }
}
